Private/Utilities.ps1

```diff
@@ -0,0 +1,38 @@
+#Requires -PSEdition Core -Version 6
+
+function Get-TempPath {
```

Comment:
nit: there is method for this. finding docs

Reply:
`Get-PSDrive Temp`

Though only available in PS Core 7+

Reply:
Yeah. Tried `[system.io.path]::GetTempPath()` but only works on Windows even if .Net core is installed on Linux.